api-client/src/main/java/com/takipi/api/client/request/team/UpdateTeamMembersRequest.java

```diff
@@ -0,0 +1,100 @@
+package com.takipi.api.client.request.team;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.Lists;
+import com.google.gson.Gson;
+import com.takipi.api.client.data.team.ServiceUsers;
+import com.takipi.api.client.data.team.ServiceUsers.TeamMember;
+import com.takipi.api.client.data.team.ServiceUsersResponse;
+import com.takipi.api.client.request.ServiceRequest;
+import com.takipi.api.core.request.intf.ApiPostRequest;
+
+import java.util.List;
+
+public class UpdateTeamMembersRequest extends ServiceRequest implements ApiPostRequest<ServiceUsersResponse>

```

Comment:
If this request simply adds members, and doesn't actually let you update any new information, I believe `AddTeamMembersRequest` is a clearer name